README.md

```diff
@@ -2,25 +2,64 @@
 
 # operatorkit
 
-The operatorkit package implements an opinionated framework for developing
-Kubernetes operators. It can be used as a library in golang. It is built on
-top of our [microkit](https://github.com/giantswarm/microkit) framework which
-provides base functionality like logging and error handling.
+The operatorkit is a library for creating [Kubernetes operators][operators]. It
+emerged as extracted common functionality of number of the operators we
+developed in Giant Swarm. The goal of the library is to provide common
+structure of the operator projects and encapsulate best practices learned
+during running operators in production.
+
+## Features
+
+- Reducing boilerplate code.
+- CRD/TPR primitives allowing to reliably create, watch custom resources, and
+  decode custom objects.
+- Independent packages. It is possible to use only a part of the library. 
+- Possibility to change behaviour that often is specific to an organization
+  like logging and error handling.
+- Minimal set of dependencies.
```

Comment:
The reconciler framework is not listed. It provides primitives to reconcile arbitrary resources which is one if the best features IMO.

Reply:
The deterministic informer is not listed. This is something the community does not provide so far. I like this feature a lot as well.

Reply:
I mentioned it in the `Current Scope`. I proposed removing `Features` in other comment.